pulp_rpm/app/models/repository.py

```diff
@@ -51,6 +51,12 @@ class RpmRemote(Remote):
     TYPE = "rpm"
     sles_auth_token = models.CharField(max_length=512, null=True)
 
+    uln_user = models.CharField(max_length=512, null=True)
+
+    uln_pwd = models.CharField(max_length=512, null=True)
```

Comment:
I noticed there are `username` and `password` fields in the `Remote` base class from pulpcore.
My question is if those are in use for anything specific, or could we repurpose them for this change (instead of adding `uln_user` and `uln_pwd`)?

Also, why are they part of the `Remote` base class? Are they really needed for every type of remote in every plugin?!

Reply:
@dkliban I wonder if you could give some input on this question. Given I just watched a `headers` field being added to the `remote` base class, I guess this base class contains anything that might be of interest to more than one type of remote. I presume the existing `username` and `password` are intended in the same way, and we could have users use those _instead_ of adding `uln_user` and `uln_pwd`. Is that right?

Reply:
Done.